java_console/ui/build.gradle

```diff
@@ -21,9 +21,13 @@ dependencies {
 }
 
 shadowJar {
+    dependsOn(':enum_to_string:shadowJar')
+    dependsOn(':config_definition:shadowJar')
     archivesBaseName = 'fome_console'
-    destinationDir = file("$rootDir/../java_console_binary")
-    archiveClassifier = ''
+    base {
+        archivesBaseName = "${rootDir}/../java_console_binary/rusefi_console"
```

Comment:
wrong binary name

Reply:
also, does this not get specified on the previous line?

Reply:
@mck1117 try now. I guess the point is to specify destinationDir in a new way